src/config/configSchema.ts

```diff
@@ -281,6 +282,11 @@ export const configSchema = {
     type: 'integer',
     default: 2,
   },
+  recordMopChallengeModes: {
```

Comment:
This should probably just be named "recordChallengeModes" - guessing we'll have WOD challenge modes soon